meson.build

```diff
@@ -2215,6 +2216,16 @@ if conf.get('ENABLE_NETWORKD', false)
                    install_dir : rootbindir)
         public_programs += [exe]
 endif
+
+exe = executable('tbtctl',
```

Comment:
Could we consider a different name? E.g. `boltctl` or even `bolt`. A quick repoquery suggests that neither prefix is currently used in Fedora.

Reply:
I am totally open to that.